src/components/map/LayersService.js

```diff
@@ -643,6 +648,8 @@ goog.require('ga_urlutils_service');
             olLayer.timestamps = config.timestamps;
             olLayer.geojsonUrl = config.geojsonUrl;
             olLayer.updateDelay = config.updateDelay;
+            olLayer.externalStyleUrl = opts && opts.externalStyleUrl ?
+                opts.externalStyleUrl : null;
```

Comment:
can't we take `styleUrl` here?

Reply:
As we already have a `stlyeUrl` property on the geojson layers at the layers configuration level, I found it confusing to use the same word for native and external styles.  In theory it would be possible to switch between the native style and the external one, with this approach we can still do that.